hw/doc/cores.md

```diff
@@ -3,7 +3,7 @@
 Cores in OpenTitan are processing units that can run programs.
 
 Currently, there are two cores in OpenTitan:
-* [Ibex](../ip/rv_core_ibex/README.md) (RV32IMCB)
+* [Ibex](../top_earlgrey/ip_autogen/rv_core_ibex/README.md) (RV32IMCB)
```

Comment:
This again raises the question about how we refer/link to ipgen'd HW IP blocks in the documentation. So far we seem to default to Earlgrey, which in a way makes sense at the moment given it's the more mature top-level design. But what do we do in the longer term?

We don't have to solve this in this PR, but do we have an issue to discuss and track this?

Reply:
Valid point. I followed previous approaches. I don't think that there is an issue for that.

Reply:
I think this falls under https://github.com/lowRISC/opentitan/issues/25721.